conf/modules.config

```diff
@@ -239,7 +248,16 @@ process {
         ]
     }
 
-    withName: KRONA {
+    withName: KREPORT2KRONA_CENTRIFUGE {
+        publishDir = [
+            // path: { "${params.outdir}/Taxonomy/${meta.classifier}/${meta.id}" },
```

Comment:
Is this meant to be commented out?

Reply:
Oops forgot to uncommend after setting enabled: fasle!